tests/locators_dashboard_page.py

```diff
@@ -65,3 +74,7 @@ class URLLocators:
     URL_NEW_VIEW = 'http://localhost:8080/newView'
     URL_FOOTER_REST_API = 'http://localhost:8080/api/'
     URL_FOOTER_VERSION = 'https://www.jenkins.io/'
+
+class Titles:
+    TITLE_DASHBOARD_PAGE = 'Dashboard [Jenkins]'
+    TITLE_NEW_ITEM = 'New Item [Jenkins]'
```

Comment:
I assume this Title belongs to different page. That means it shouldn't be here.

Reply:
ok, I'll change it

Reply:
now this test is skipping.